docs/current_docs/integrations/838502-php.mdx

```diff
@@ -0,0 +1,95 @@
+---
+slug: /integrations/838502/php
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# PHP
+
+Dagger can be used to perform common CI tasks - testing, containerizing, publishing and more - for any PHP application.
+
+The following code sample demonstrates how these CI tasks can be encapsulated as Dagger Functions in a Dagger module. It assumes:
+
+- A PHP 8.2.x Web application with:
+  - Composer for package management
+  - PHPUnit for application testing
+- Credentials to publish the containerized application image to a registry
+
+<Tabs groupId="language">
```

Comment:
Might be helpful, since this documentation is for deploying a PHP app to include a code sample for doing it using the PHP SDK as well.

Reply:
With the release of Dagger Functions, using an SDK is more of a niche case eg. you would use the SDK when you prefer to build a custom tool over using the Dagger CLI. With this in mind, we're focusing on Functions code rather than SDK code atm. See also the discussion at https://discord.com/channels/707636530424053791/1120503349599543376/1225515198199234560